.github/workflows/tests.yml

```diff
@@ -8,6 +8,14 @@ jobs:
     steps:
       - name: Clone the repository
         uses: actions/checkout@v2
+      - name: Cache Cargo artifacts
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
```

Comment:
I wouldn't be surprised if tilde expansion wouldn't be performed on these paths.